src/io/flutter/sdk/FlutterSdk.java

```diff
@@ -321,7 +322,7 @@ public FlutterCommand flutterTest(@NotNull PubRoot root, @NotNull VirtualFile fi
         throw new IllegalStateException("Flutter SDK is too old to debug tests");
       }
     }
-    if (mode == RunMode.DEBUG || mode == RunMode.RUN) {
```

Comment:
add a comment on why this check is here with a link to a bug. Otherwise the next person looking at this code will be confused.